database/fixtures.sql

```diff
@@ -0,0 +1,193 @@
+BEGIN;
+
+
+INSERT INTO "users" (username, email, password)
+VALUES
+  ('DanielEineving', 'daniel@eineving.se', 'danieleineving'),
+  ('JimmyBerlin', 'berlin.jimmy@gmail.com', 'jimmyberlin'),
+  ('MikaelLonn', 'mickelonn93@gmail.com', 'mikaellonn'),
+  ('ChristopherAkersten', 'christopher.akersten@gmail.com', 'christopherakersten'),
+  ('JohanAndersson', 'lol2kpe@gmail.com', 'johanandersson');
+
+COMMIT;
+
+INSERT INTO token (token, userid)
+VALUES ('testtoken', (SELECT id
+                      FROM "users"
+                      WHERE username = 'ChristopherAkersten'
+                      LIMIT 1));
+
+COMMIT;
+
+INSERT INTO customer (name, email) VALUES ('Testbolag', 'test@testbolag.se');
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=plsql%3AS1192)